JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -228,8 +228,8 @@ public function denormalize($data, $class, $format = null, array $context = [])
         );
 
         foreach ($normalizedData as $attributeName => $attributeValue) {
-            // Ignore JSON-LD special attributes
-            if ('@' === $attributeName[0]) {
+            // Ignore JSON-LD special attributes and null attribute names
+            if ('@' === $attributeName[0] || is_null($attributeName[0])) {
```

Comment:
@dylanjacob is it still relevant to you? Like @dunglas, I would like to see why this check is required... Moreover, can you please write a test?